app/keepers/keepers.go

```diff
@@ -721,6 +704,11 @@ func (ak *AppKeepers) InitKeepers(
 	// Setting Router will finalize all routes by sealing router
 	// No more routes can be added
 	ak.IBCKeeper.SetRouter(ibcRouter)
+
+	// Create IBCv2 Router & seal
+	ibcv2Router := ibcapi.NewRouter().
+		AddRoute(ibctransfertypes.PortID, transferStackV2)
+	ak.IBCKeeper.SetRouterV2(ibcv2Router)
```

Comment:
Is L696-702 still needed?

Reply:
Yes same thing as transfer stack, no mention in migration guide [and gaia has both](https://github.com/cosmos/gaia/blob/abdbacbb3c53db29c9ff57531f4069ad59ed3754/app/keepers/keepers.go#L566)